### PR TITLE
Bug fixes

### LINKED ISSUE #58: 'broken' coral should be 'bleached'
The use of "fragments" does not imply 'broken'; it's a domain term for referencing each individual coral.  We're not concerned with "broken" ... we are, however, concerned with the number of `bleached` coral; update the forms and the database columns accordingly.

### PR DESCRIPTION
**Pull request by conj40**
### Description

Updated the forms and the database columns for restoration_activity_log_entries from "broken_columns" to "bleached_columns" I also updated the html accordingly.

### Type of change

* Bug fix (A non-breaking change which fixes an issue)

### How Has This Been Tested?

Tested on my local machine and no problems.

### CODE REVIEW

**Comment by cflipse** [comment]
We very much do not want schema.rb excluded from the repository; please rollback that particular commit.

Otherwise, looks fine, once tests go green

Thanks!

**Comment by conj40** [comment]
> We very much do not want schema.rb excluded from the repository; please rollback that particular commit.
> 
> Otherwise, looks fine, once tests go green
> 
> Thanks!

Hang on will update  - I have just realised the error on robo cop :)

**Comment by Chub93** [comment]
> We

**Comment by Chub93** [comment]
Ah failed reasons are robo cop for active_storage_attachments

File: rails/db/migrate/20190726200834_create_restoration_activity_log_entries.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ def change
     create_table :restoration_activity_log_entries do |t|
       t.boolean :cleaned
       t.integer :percent_filled
-      t.integer :broken_corals
+      t.integer :broken_columns
```
Comment (cflipse) [inline rails/db/migrate/20190726200834_create_restoration_activity_log_entries.rb]: Looks like an accidental change to an old migration; please drop this one.

File: rails/db/migrate/20190813201251_rename_restoration_column.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class RenameRestorationColumn < ActiveRecord::Migration[5.2]
+  def change
+    rename_column :restoration_activity_log_entries, :broken_columns, :bleached_corals
```
Comment (cflipse) [inline rails/db/migrate/20190813201251_rename_restoration_column.rb]: And use the correct name to migrate away from here.

**Pull request merged**